apps/web/src/data/ecosystem.json

```diff
@@ -3654,5 +3654,13 @@
     "description": "Virtuals Protocol is building a Society of AI Agents onchain",
     "url": "https://app.virtuals.io/",
     "imageUrl": "/images/partners/virtuals.png"
+  },
+  {
+    "name": "Metacade",
+    "category": "consumer",
+    "subcategory": "gaming",
+    "description": "The Best Place to Play, Build, Connect and Earn Onchain.",
```

Comment:
@soheimam 

Can you please drop the superlative, "The Best Place to" and change to: "Play Build, Connect and Earn Onchain."